gridstatus/ieso.py

```diff
@@ -844,3 +911,883 @@ def _request(self, url, verbose):
             )
 
         return r
+
+    def get_resource_adequacy_report(
```

Comment:
Do we want to decorate this with `@support_date_range`? We can use `@support_date_range(frequency="DAY_START")` then the function should work for date ranges.

Reply:
definitely. added along with some new tests to cover the ranges and vintage options.